diff --git a/node_modules/react-native-text-recognition/android/build.gradle b/node_modules/react-native-text-recognition/android/build.gradle
index b76e895..abd65eb 100644
--- a/node_modules/react-native-text-recognition/android/build.gradle
+++ b/node_modules/react-native-text-recognition/android/build.gradle
@@ -18,11 +18,11 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('TextRecognition_compileSdkVersion', 29)
+    compileSdkVersion 33
     buildToolsVersion safeExtGet('TextRecognition_buildToolsVersion', '29.0.2')
     defaultConfig {
         minSdkVersion safeExtGet('TextRecognition_minSdkVersion', 16)
-        targetSdkVersion safeExtGet('TextRecognition_targetSdkVersion', 29)
+        targetSdkVersion 33
         versionCode 1
         versionName "1.0"
 
